static/app/components/events/interfaces/spans/embeddedSpanTree.tsx

```diff
@@ -72,12 +72,12 @@ export function EmbeddedSpanTree(props: Props) {
         api={api}
         location={location}
       >
-        {_results => {
-          if (_results.isLoading) {
+        {results => {
+          if (results.isLoading) {
```

Comment:
Not related to the PR, but now that we're no longer using a nested query here we can just call this `results`